final_html_output.py

```diff
@@ -32,3 +33,7 @@ def final_html_output(df: pd.DataFrame) -> str:
 
 def link_username(u: str):
     return f'<a target="_blank" rel="noopener noreferrer" href="https://github.com/{u}">{u}</a>'
+
+
+def set_user_avatar(url: str):
+    return f'<img src="{url}" />'
```

Comment:
Let's set the width and the height attribute to 20 as well. Also, set the alt attribute to `@{username}`. GitHub does the same.

Reply:
For semantic and accessibilty reasons.